### ISSUE TITLE
Compiling on debian testing

### ISSUE THREAD

**Issue by tangb** [plexinc/plex-media-player#660]
Hi

I'm compiling PMP for a while now and for 3-4 weeks I got this error:

> ERROR: Conanfile: 'settings.compiler' value not defined

My computer is running debian testing, it is up to date, conan is updated too (pip install -U conan).
I don't find useful infos on google about this problem.

Can you help me ?

Thank you.

Tang

**Comment by tangb** [comment]
I just tested now and got no problem with conan since it is removed from build process.
Now everything is compiling fine but I got black screen when launching plex media player.
Below the log .local/share/plexmediaplayer/logs/plexmediaplayer.log.
If you need more info, I can help.

Thank you

```
2018-03-07 22:59:25 [ INFO  ] Log.cpp @ 115 - Starting Plex Media Player version: 2.6.0.dev-edcf527b build date: 2018-03-07 
2018-03-07 22:59:25 [ INFO  ] Log.cpp @ 116 -   Running on: Debian GNU/Linux buster/sid [4.14.0-3-amd64] arch x86_64 
2018-03-07 22:59:25 [ INFO  ] Log.cpp @ 117 -   Qt Version: 5.7.1 [x86_64-little_endian-lp64] 
2018-03-07 22:59:25 [ DEBUG ] UpdateManager.cpp @ 45 - No Update directory found, exiting 
2018-03-07 22:59:26 [ WARN  ] Log.cpp @ 35 - "QObject::connect: invalid null parameter" 
2018-03-07 22:59:26 [ ERROR ] SettingsComponent.cpp @ 263 - Trying to load section: "openelec" from config file, but we don't want that. 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: settings inited 
2018-03-07 22:59:26 [ DEBUG ] HTTPServer.cpp @ 32 - Listening to port: 32433 
2018-03-07 22:59:26 [ INFO  ] InputMapping.cpp @ 113 - Loading inputmaps from: ":/inputmaps" 
2018-03-07 22:59:26 [ WARN  ] InputMapping.cpp @ 92 - Missing element 'idmatcher' from mapping file: ":/inputmaps/dualshock4-xbox-emulate.json" 
2018-03-07 22:59:26 [ INFO  ] InputMapping.cpp @ 113 - Loading inputmaps from: "/root/.local/share/plexmediaplayer/inputmaps" 
2018-03-07 22:59:26 [ INFO  ] InputComponent.cpp @ 54 - Successfully inited input: Keyboard 
2018-03-07 22:59:26 [ INFO  ] LocalJsonServer.cpp @ 33 - Listening to socket: "/tmp/pmp_inputSocket_root.sock" 
2018-03-07 22:59:26 [ INFO  ] InputComponent.cpp @ 54 - Successfully inited input: socket 
2018-03-07 22:59:26 [ INFO  ] InputComponent.cpp @ 54 - Successfully inited input: roku 
2018-03-07 22:59:26 [ INFO  ] InputSDL.cpp @ 207 - SDL found  0  joysticks 
2018-03-07 22:59:26 [ INFO  ] InputComponent.cpp @ 54 - Successfully inited input: SDL 
2018-03-07 22:59:26 [ ERROR ] InputLIRC.cpp @ 82 - LIRC Socket Error :  QLocalSocket::ServerNotFoundError 
2018-03-07 22:59:26 [ WARN  ] InputComponent.cpp @ 50 - Failed to init input: LIRC 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: input inited 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: system inited 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 20 - "DisplayManager found 1 Display(s)." 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 26 - "Available modes for Display #0 (HDMI-0)" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  0:  1920 x 1080 x  0bpp @60Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  1:  1920 x 1080 x  0bpp @59.9394Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  2:  1920 x 1080 x  0bpp @50Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  3:  1920 x 1080ix  0bpp @59.9968Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  4:  1920 x 1080ix  0bpp @50.0445Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  5:  1680 x 1050 x  0bpp @59.9543Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  6:  1440 x  900 x  0bpp @74.9844Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  7:  1440 x  900 x  0bpp @59.8874Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  8:  1280 x 1024 x  0bpp @75.0247Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode  9:  1280 x 1024 x  0bpp @60.0197Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 10:  1280 x  720 x  0bpp @60Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 11:  1280 x  720 x  0bpp @59.9434Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 12:  1280 x  720 x  0bpp @50Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 13:  1024 x  768 x  0bpp @75.0286Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 14:  1024 x  768 x  0bpp @60.0038Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 15:   800 x  600 x  0bpp @75Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 16:   800 x  600 x  0bpp @60.3165Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 17:   720 x  576 x  0bpp @50Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 18:   720 x  576ix  0bpp @50.0801Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 19:   720 x  480 x  0bpp @59.9401Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 20:   720 x  480ix  0bpp @60.0545Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 21:   640 x  480 x  0bpp @75Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 22:   640 x  480 x  0bpp @72.8088Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 23:   640 x  480 x  0bpp @59.9405Hz" 
2018-03-07 22:59:26 [ INFO  ] DisplayManager.cpp @ 30 - "Mode 24:   640 x  480 x  0bpp @59.9286Hz" 
2018-03-07 22:59:26 [ ERROR ] DisplayManager.cpp @ 47 - DisplayManager : unable to retrieve main display 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: display inited 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: updater inited 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: remote inited 
2018-03-07 22:59:26 [ DEBUG ] SettingsComponent.cpp @ 140 - Updated possible values for: "device" to (QVariant(QVariantMap, QMap(("title", QVariant(QString, "Autoselect device"))("value", QVariant(QString, "auto")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "Built-in Audio Analog Stereo"))("value", QVariant(QString, "pulse/alsa_output.pci-0000_00_08.0.analog-stereo")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "Default (alsa)"))("value", QVariant(QString, "alsa")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/Default Audio Device"))("value", QVariant(QString, "alsa/sysdefault:CARD=NVidia")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/Front speakers"))("value", QVariant(QString, "alsa/front:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/2.1 Surround output to Front and Subwoofer speakers"))("value", QVariant(QString, "alsa/surround21:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/4.0 Surround output to Front and Rear speakers"))("value", QVariant(QString, "alsa/surround40:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/4.1 Surround output to Front, Rear and Subwoofer speakers"))("value", QVariant(QString, "alsa/surround41:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/5.0 Surround output to Front, Center and Rear speakers"))("value", QVariant(QString, "alsa/surround50:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/5.1 Surround output to Front, Center, Rear and Subwoofer speakers"))("value", QVariant(QString, "alsa/surround51:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/7.1 Surround output to Front, Center, Side, Rear and Woofer speakers"))("value", QVariant(QString, "alsa/surround71:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Digital/IEC958 (S/PDIF) Digital Audio Output"))("value", QVariant(QString, "alsa/iec958:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, HDMI 0/HDMI Audio Output"))("value", QVariant(QString, "alsa/hdmi:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/Direct sample mixing device"))("value", QVariant(QString, "alsa/dmix:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Digital/Direct sample mixing device"))("value", QVariant(QString, "alsa/dmix:CARD=NVidia,DEV=1")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, HDMI 0/Direct sample mixing device"))("value", QVariant(QString, "alsa/dmix:CARD=NVidia,DEV=3")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Analog/Hardware device with all software conversions"))("value", QVariant(QString, "alsa/plughw:CARD=NVidia,DEV=0")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, ALC888 Digital/Hardware device with all software conversions"))("value", QVariant(QString, "alsa/plughw:CARD=NVidia,DEV=1")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "HDA NVidia, HDMI 0/Hardware device with all software conversions"))("value", QVariant(QString, "alsa/plughw:CARD=NVidia,DEV=3")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "Default (jack)"))("value", QVariant(QString, "jack")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "Default (sdl)"))("value", QVariant(QString, "sdl")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "Default (sndio)"))("value", QVariant(QString, "sndio"))))) 
2018-03-07 22:59:26 [ DEBUG ] PlayerComponent.cpp @ 921 - Audio devices removed: QSet() 
2018-03-07 22:59:26 [ DEBUG ] PlayerComponent.cpp @ 922 - Audio devices added: QSet("alsa/plughw:CARD=NVidia,DEV=0", "alsa/surround50:CARD=NVidia,DEV=0", "alsa/plughw:CARD=NVidia,DEV=1", "alsa/surround21:CARD=NVidia,DEV=0", "alsa/plughw:CARD=NVidia,DEV=3", "alsa/surround71:CARD=NVidia,DEV=0", "alsa/sysdefault:CARD=NVidia", "alsa/dmix:CARD=NVidia,DEV=0", "alsa/surround41:CARD=NVidia,DEV=0", "auto", "alsa/dmix:CARD=NVidia,DEV=1", "alsa", "alsa/iec958:CARD=NVidia,DEV=0", "alsa/dmix:CARD=NVidia,DEV=3", "alsa/surround40:CARD=NVidia,DEV=0", "alsa/front:CARD=NVidia,DEV=0", "sdl", "alsa/hdmi:CARD=NVidia,DEV=0", "alsa/surround51:CARD=NVidia,DEV=0", "pulse/alsa_output.pci-0000_00_08.0.analog-stereo", "jack", "sndio") 
2018-03-07 22:59:26 [ DEBUG ] PlayerComponent.cpp @ 923 - Audio device selected: "auto" 
2018-03-07 22:59:26 [ INFO  ] PlayerComponent.cpp @ 1078 - Audio Config - device: auto, channel layout: 2.0, passthrough codecs: none, ac3 transcoding: no 
2018-03-07 22:59:26 [ INFO  ] PlayerComponent.cpp @ 1215 - Codec "mpeg2video" present 
2018-03-07 22:59:26 [ INFO  ] PlayerComponent.cpp @ 1215 - Codec "vc1" present 
2018-03-07 22:59:26 [ INFO  ] PlayerComponent.cpp @ 219 - Present codecs: aasc aic alias_pix amv anm ansi apng asv1 asv2 aura aura2 avrp avrn avs avui ayuv bethsoftvid bfi binkvideo bmp bmv_video brender_pix c93 cavs cdgraphics cdxl cfhd cinepak clearvideo cljr cllc cpia camstudio cyuv dds dfa dirac dnxhd dpx dsicinvideo dvvideo dxa dxtory dxv eacmv eamad eatgq eatgv eatqi 8bps escape124 escape130 exr ffv1 ffvhuff fic fits flashsv flashsv2 flic flv fmvc 4xm fraps frwu g2m gdv gif h261 h263 h263i h263p h263_v4l2m2m h264 h264_crystalhd h264_v4l2m2m hap hevc hnm4video hq_hqa hqx huffyuv idcinvideo iff indeo2 indeo3 indeo4 indeo5 interplayvideo jpeg2000 jpegls jv kgv1 kmvc lagarith loco m101 magicyuv mdec mimic mjpeg mjpegb mmvideo motionpixels mpegvideo_xvmc mpeg1video mpeg2video mpeg4 mpeg4_crystalhd mpeg4_v4l2m2m mpegvideo mpeg1_v4l2m2m mpeg2_crystalhd mpeg2_v4l2m2m msa1 mscc msmpeg4v1 msmpeg4v2 msmpeg4 msmpeg4_crystalhd msrle mss1 mss2 msvideo1 mszh mts2 mvc1 mvc2 mxpeg nuv paf_video pam pbm pcx pgm pgmyuv pictor pixlet png ppm prores prores_lgpl psd ptx qdraw qpeg qtrle r10k r210 rawvideo rl2 roqvideo rpza rscc rv10 rv20 rv30 rv40 sanm scpr screenpresso sgi sgirle sheervideo smackvid smc smvjpeg snow sp5x speedhq srgc sunrast svq1 svq3 targa targa_y216 tdsc theora thp tiertexseqvideo tiff tmv truemotion1 truemotion2 truemotion2rt camtasia tscc2 txd ultimotion utvideo v210 v210x v308 v408 v410 vb vble vc1 vc1_crystalhd vc1image vc1_v4l2m2m vcr1 vmdvideo vmnc vp3 vp5 vp6 vp6a vp6f vp7 vp8 vp8_v4l2m2m vp9 vp9_v4l2m2m vqavideo bitpacked webp wrapped_avframe wmv1 wmv2 wmv3 wmv3_crystalhd wmv3image wnv1 xan_wc3 xan_wc4 xbm xface xl xpm xwd y41p ylc yop yuv4 012v zerocodec zlib zmbv libopenjpeg librsvg libvpx libvpx-vp9 bintext xbin idf h264_cuvid hevc_cuvid mjpeg_cuvid mpeg1_cuvid mpeg2_cuvid mpeg4_cuvid vc1_cuvid vp8_cuvid vp9_cuvid comfortnoise dvaudio 8svx_exp 8svx_fib s302m sdx2_dpcm aac aac_fixed aac_latm ac3 ac3_fixed alac als amrnb amrwb ape atrac1 atrac3 atrac3al atrac3plus atrac3plusal binkaudio_dct binkaudio_rdft bmv_audio cook dca dolby_e dsd_lsbf dsd_msbf dsd_lsbf_planar dsd_msbf_planar dsicinaudio dss_sp dst eac3 evrc wavesynth flac g723_1 g729 gsm gsm_ms iac imc interplayacm mace3 mace6 metasound mlp mp1 mp1float mp2 mp2float mp3 mp3float mp3adu mp3adufloat mp3on4 mp3on4float mpc7 mpc8 nellymoser on2avc opus paf_audio qcelp qdm2 qdmc real_144 real_288 ralf shorten sipr smackaud sonic tak truehd truespeech tta twinvq vmdaudio vorbis wavpack wmalossless wmapro wmav1 wmav2 wmavoice ws_snd1 xma1 xma2 pcm_alaw pcm_bluray pcm_dvd pcm_f16le pcm_f24le pcm_f32be pcm_f32le pcm_f64be pcm_f64le pcm_lxf pcm_mulaw pcm_s8 pcm_s8_planar pcm_s16be pcm_s16be_planar pcm_s16le pcm_s16le_planar pcm_s24be pcm_s24daud pcm_s24le pcm_s24le_planar pcm_s32be pcm_s32le pcm_s32le_planar pcm_s64be pcm_s64le pcm_u8 pcm_u16be pcm_u16le pcm_u24be pcm_u24le pcm_u32be pcm_u32le pcm_zork gremlin_dpcm interplay_dpcm roq_dpcm sol_dpcm xan_dpcm adpcm_4xm adpcm_adx adpcm_afc adpcm_aica adpcm_ct adpcm_dtk adpcm_ea adpcm_ea_maxis_xa adpcm_ea_r1 adpcm_ea_r2 adpcm_ea_r3 adpcm_ea_xas g722 g726 g726le adpcm_ima_amv adpcm_ima_apc adpcm_ima_dat4 adpcm_ima_dk3 adpcm_ima_dk4 adpcm_ima_ea_eacs adpcm_ima_ea_sead adpcm_ima_iss adpcm_ima_oki adpcm_ima_qt adpcm_ima_rad adpcm_ima_smjpeg adpcm_ima_wav adpcm_ima_ws adpcm_ms adpcm_mtaf adpcm_psx adpcm_sbpro_2 adpcm_sbpro_3 adpcm_sbpro_4 adpcm_swf adpcm_thp adpcm_thp_le adpcm_vima adpcm_xa adpcm_yamaha libgsm libgsm_ms libopus libspeex libvorbis a64multi(enc) a64multi5(enc) alias_pix(enc) amv(enc) apng(enc) asv1(enc) asv2(enc) avrp(enc) avui(enc) ayuv(enc) bmp(enc) cinepak(enc) cljr(enc) comfortnoise(enc) dnxhd(enc) dpx(enc) dvvideo(enc) ffv1(enc) ffvhuff(enc) fits(enc) flashsv(enc) flashsv2(enc) flv(enc) gif(enc) h261(enc) h263(enc) h263p(enc) hap(enc) huffyuv(enc) jpeg2000(enc) jpegls(enc) ljpeg(enc) mjpeg(enc) mpeg1video(enc) mpeg2video(enc) mpeg4(enc) msmpeg4v2(enc) msmpeg4(enc) msvideo1(enc) pam(enc) pbm(enc) pcx(enc) pgm(enc) pgmyuv(enc) png(enc) ppm(enc) prores(enc) prores_aw(enc) prores_ks(enc) qtrle(enc) r10k(enc) r210(enc) rawvideo(enc) roqvideo(enc) rv10(enc) rv20(enc) s302m(enc) sgi(enc) snow(enc) sunrast(enc) svq1(enc) targa(enc) tiff(enc) utvideo(enc) v210(enc) v308(enc) v408(enc) v410(enc) vc2(enc) wrapped_avframe(enc) wmv1(enc) wmv2(enc) xbm(enc) xface(enc) xwd(enc) y41p(enc) yuv4(enc) zlib(enc) zmbv(enc) aac(enc) ac3(enc) ac3_fixed(enc) alac(enc) dca(enc) eac3(enc) flac(enc) g723_1(enc) mlp(enc) mp2(enc) mp2fixed(enc) nellymoser(enc) opus(enc) real_144(enc) sonic(enc) sonicls(enc) truehd(enc) tta(enc) vorbis(enc) wavpack(enc) wmav1(enc) wmav2(enc) pcm_alaw(enc) pcm_f32be(enc) pcm_f32le(enc) pcm_f64be(enc) pcm_f64le(enc) pcm_mulaw(enc) pcm_s8(enc) pcm_s8_planar(enc) pcm_s16be(enc) pcm_s16be_planar(enc) pcm_s16le(enc) pcm_s16le_planar(enc) pcm_s24be(enc) pcm_s24daud(enc) pcm_s24le(enc) pcm_s24le_planar(enc) pcm_s32be(enc) pcm_s32le(enc) pcm_s32le_planar(enc) pcm_s64be(enc) pcm_s64le(enc) pcm_u8(enc) pcm_u16be(enc) pcm_u16le(enc) pcm_u24be(enc) pcm_u24le(enc) pcm_u32be(enc) pcm_u32le(enc) roq_dpcm(enc) adpcm_adx(enc) g722(enc) g726(enc) g726le(enc) adpcm_ima_qt(enc) adpcm_ima_wav(enc) adpcm_ms(enc) adpcm_swf(enc) adpcm_yamaha(enc) ssa(enc) ass(enc) dvbsub(enc) dvdsub(enc) mov_text(enc) srt(enc) subrip(enc) text(enc) webvtt(enc) xsub(enc) libgsm(enc) libgsm_ms(enc) libmp3lame(enc) libopenjpeg(enc) libopus(enc) libshine(enc) libspeex(enc) libtheora(enc) libtwolame(enc) libvorbis(enc) libvpx(enc) libvpx-vp9(enc) libwavpack(enc) libwebp_anim(enc) libwebp(enc) libx264(enc) libx264rgb(enc) libx265(enc) libxvid(enc) h263_v4l2m2m(enc) h264_nvenc(enc) h264_omx(enc) h264_v4l2m2m(enc) h264_vaapi(enc) nvenc(enc) nvenc_h264(enc) nvenc_hevc(enc) hevc_nvenc(enc) hevc_vaapi(enc) mjpeg_vaapi(enc) mpeg2_vaapi(enc) mpeg4_v4l2m2m(enc) vp8_v4l2m2m(enc) vp8_vaapi(enc) vp9_vaapi(enc) 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: player inited 
2018-03-07 22:59:26 [ INFO  ] ComponentManager.cpp @ 41 - Component: power inited 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: switch mapped to DisplayComponent::switchCommand 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 264 - Host command maps to method with an argument. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: recreateRpiUI mapped to DisplayManagerX11::resetRendering 
2018-03-07 22:59:26 [ ERROR ] InputComponent.cpp @ 273 - Slot for host command missing, or has incorrect signature! 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: player mapped to PlayerComponent::userCommand 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 264 - Host command maps to method with an argument. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: poweroff mapped to PowerComponent::PowerOff 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: reboot mapped to PowerComponent::Reboot 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: suspend mapped to PowerComponent::Suspend 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: cycle_setting mapped to SettingsComponent::cycleSettingCommand 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 264 - Host command maps to method with an argument. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: set_setting mapped to SettingsComponent::setSettingCommand 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 264 - Host command maps to method with an argument. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: crash! mapped to SystemComponent::crashApp 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: script mapped to SystemComponent::runUserScript 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 264 - Host command maps to method with an argument. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: message mapped to SystemComponent::hostMessage 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 264 - Host command maps to method with an argument. 
2018-03-07 22:59:26 [ DEBUG ] SettingsComponent.cpp @ 783 - () 
2018-03-07 22:59:26 [ DEBUG ] HelperLauncher.cpp @ 52 - Connecting to helper 
2018-03-07 22:59:26 [ DEBUG ] HelperLauncher.cpp @ 100 - Failed to connect to helper: "QLocalSocket::connectToServer: Invalid name" 
2018-03-07 22:59:26 [ DEBUG ] HelperLauncher.cpp @ 158 - Launching helper: "/usr/local/bin/pmphelper" 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: close mapped to KonvergoWindow::close 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: toggleDebug mapped to KonvergoWindow::toggleDebug 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: reload mapped to KonvergoWindow::reloadWeb 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: fullscreen mapped to KonvergoWindow::toggleFullscreen 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: minimize mapped to KonvergoWindow::minimizeWindow 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 255 - Adding host command: fullscreenCurrentMode mapped to KonvergoWindow::toggleFullscreenNoSwitch 
2018-03-07 22:59:26 [ DEBUG ] InputComponent.cpp @ 269 - Host command maps to method without arguments. 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 276 - Restoring geo: QRect(320,180 1280x720) 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 198 - Window state when saving geometry: QWindow::Visibility(Hidden) 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 209 - Saving window geometry: QRect(320,180 1280x720) 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 513 - QWindow visibility set to QWindow::Visibility(Windowed) 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 276 - Restoring geo: QRect(320,180 1280x720) 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 198 - Window state when saving geometry: QWindow::Visibility(Windowed) 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 209 - Saving window geometry: QRect(320,180 1280x720) 
2018-03-07 22:59:26 [ DEBUG ] KonvergoWindow.cpp @ 732 - Screen 0 QScreen(0x562ce8618520, name="HDMI-0") QRect(0,0 1920x1080) QRect(0,0 1920x1080) active: true selected: false 
2018-03-07 22:59:26 [ DEBUG ] SettingsComponent.cpp @ 140 - Updated possible values for: "forceFSScreen" to (QVariant(QVariantMap, QMap(("title", QVariant(QString, "Auto"))("value", QVariant(QString, "")))), QVariant(QVariantMap, QMap(("title", QVariant(QString, "0,0 1919x1079 (HDMI-0) *"))("value", QVariant(QString, "HDMI-0"))))) 
```

**Comment by LongChair** [comment]
conan isn't used in the build anymore for now, closing this.
